development/prow-installer/pkg/k8s/client.go

```diff
@@ -0,0 +1,84 @@
+package k8s
+
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"google.golang.org/api/container/v1"
+	//"google.golang.org/api/option"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"log"
+	//"os"
+	"time"
+
+	//"github.com/kyma-project/test-infra/development/prow-installer/pkg/cluster"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+
+type API interface {
+	Get(ctx context.Context, clusterID string, zoneID string) (*container.Cluster, error)
+}
+
+//type Client struct {
+//	K8sclient *kubernetes.Clientset
+//}
+
+// Refactor prow-installer cluster package client implementation to get rid of this method. prow-installer package should be able to provide client for API interface implemented here.
+
+//func NewGKEClient (ctx context.Context, projectID string) (*cluster.APIWrapper, error) {
```

Comment:
maybe we could remove this commented out function?

Reply:
done